.claude/commands/amelia/coderabbit-respond.md

```
@@ -0,0 +1,45 @@
+description: respond to coderabbitai review comments on the current PR after evaluation and fixes
+---
+Respond to the coderabbitai[bot] review comments on the current PR.
+
+## Context
+
+You have just evaluated the CodeRabbit feedback using `/amelia:coderabbit-review` and made any necessary fixes. Now post responses to the review comments.
+
+## Steps
+
+1. **Get PR context**:
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)